tock/tock/templates/uaa_client/login_error.html

```diff
@@ -1,14 +1,7 @@
 {% extends "base.html" %}
 {% block content %}
 
-{% if not superuser %}
-<h2>Whoops, you don't have access</h2>
-
-<p>You don't have access to view this page.<p>
-{% endif %}
-
-{% if error_code %}
-<h2>Whoops, login error!</h2>
+<h2>UAA Login Error</h2>
```

Comment:
Do we really want to use the acronym "UAA" here, since this will be shown to users who might not know what it means?  Up to you, really, I just thought I'd point it out.

Reply:
I agree with @toolness. I'm an anti-acronym-ite.